app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.create(user_params)
+    if @user.save
+      flash[:notice] = "You are successfully registered and logged in!"
+      redirect_to "/profile"
+    else
+      flash[:notice] = @user.errors.full_messages.to_sentence
+      render :new
+    end
+  end
+
+  def show
+  end
+
```

Comment:
Do we need this show method at this point?

Reply:
Yes profile leads us here from the config routes

Reply:
it self-identifies as an action, not a method